### PR TITLE
Add v2.3.4 of mattermost-plugin-servicenow to the Marketplace

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
Screenshot from 2022-12-21 13-25-05

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-plugin-servicenow/issues/156

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -12816,6 +12816,214 @@
...
...pub25lOyIgZD0iTTM0NSAzMDRMMzQ2IDMwNUwzNDUgMzA0eiIvPgo8cGF0aCBzdHlsZT0iZmlsbDojYWRiNWI2OyBzdHJva2U6bm9uZTsiIGQ9Ik0zODcgMzA0TDM4OCAzMDVMMzg3IDMwNHoiLz4KPHBhdGggc3R5bGU9ImZpbGw6IzU4NmQ2Yzsgc3Ryb2tlOm5vbmU7IiBkPSJNNDA5IDMwNEw0MTAgMzA1TDQwOSAzMDRNMTk0IDMwNUwxOTUgMzA2TDE5NCAzMDVNMjA4IDMwNUwyMDkgMzA2TDIwOCAzMDVNMjcyIDMwNUwyNzMgMzA2TDI3MiAzMDVNMzIzIDMwNUwzMjQgMzA2TDMyMyAzMDVNMzg3IDMwNUwzODggMzA2TDM4NyAzMDVNNjUgMzA2TDY1IDMwN0w5MiAzMDdMNjUgMzA2TTEzOCAzMDZMMTM4IDMwN0wxNjYgMzA3TDEzOCAzMDZNMTk1IDMwNkwxOTYgMzA3TDE5NSAzMDZNMjU5IDMwNkwyNjAgMzA3TDI1OSAzMDZNMjcwIDMwNkwyNzEgMzA3TDI3MCAzMDZ6Ii8+CjxwYXRoIHN0eWxlPSJmaWxsOiNhZGI1YjY7IHN0cm9rZTpub25lOyIgZD0iTTMyNCAzMDZMMzI0IDMwN0wzNDQgMzA3TDMyNCAzMDZ6Ii8+CjxwYXRoIHN0eWxlPSJmaWxsOiM4MGIzOWY7IHN0cm9rZTpub25lOyIgZD0iTTM0NCAzMDZMMzQ1IDMwN0wzNDQgMzA2eiIvPgo8cGF0aCBzdHlsZT0iZmlsbDojYWRiNWI2OyBzdHJva2U6bm9uZTsiIGQ9Ik0zODggMzA2TDM4OCAzMDdMNDA4IDMwN0wzODggMzA2eiIvPgo8cGF0aCBzdHlsZT0iZmlsbDojNTg2ZDZjOyBzdHJva2U6bm9uZTsiIGQ9Ik00MDggMzA2TDQwOSAzMDdMNDA4IDMwNk0xOTcuNjY3IDMwNy4zMzNMMTk4LjMzMyAzMDcuNjY3TDE5Ny42NjcgMzA3LjMzM00yMDQgMzA3TDIwNSAzMDhMMjA0IDMwN00yNjEuNjY3IDMwNy4zMzNMMjYyLjMzMyAzMDcuNjY3TDI2MS42NjcgMzA3LjMzM00yNjggMzA3TDI2OSAzMDhMMjY4IDMwN3oiLz4KPC9zdmc+Cg==",
```
Comment (mickmister) [inline plugins.json:12821]: This icon data is massive. Any way we can scale this down?

Comment (mickmister) [reply plugins.json:12821]: The long scrolling really proves my point 🤣

**Comment by mickmister** [comment]
It seems the plugin was added twice in this PR. I wonder how that happened

File: plugins.json (hunk 1 above)
Comment (hanzei) [reply plugins.json:12821]: The source file (https://github.com/mattermost/mattermost-plugin-servicenow/blob/e9d9ae34f20bc52872e7ab022924b582208b4ea1/assets/icon.svg?short_path=758daa0) also contains a lot of details. Hence, does the encoding.

2/5 it's not worth spending time on reducing the size of a single entry. Caching would be the next step if the amount of data that is transferred becomes too much.

**Comment by hanzei** [comment]
@mickmister Good catch. I don't know how the duplicate entry ended up here. Fixed that.

**Comment by mickmister** [comment]
@hanzei How do you feel about adding this entry with the large icon data? It's much larger than any other entry in the `plugins.json` file

**Comment by hanzei** [comment]
My blunt answer is: It's not worth delaying the plugin release because of the larger icon. Who cares about the icon size?
 (https://github.com/mattermost/mattermost-marketplace/pull/328#discussion_r1054836923 for more details)

**Review by mickmister — DISMISSED** [review]

**Review by DHaussermann — DISMISSED** [review]
Tested and passed
- Tested install path
- Tested upgrade path but, this will not be applicable  
- Tested enabling the plugin to ensure it it starts and runs 
- Marketplace icon looks fine in light and dark theme
- Marketplace link points to the repo in the Mattermost org
- Version correctly showing as `2.3.4`

LGTM! 

Added `Do Not Merge` label until we discuss removing the older Service Now App

**Comment by hanzei** [comment]
@mickmister @DHaussermann Had to merge `production`. PTAL

**Review by DHaussermann — APPROVED** [review]
Tested and passed
Re-tested to ensured the listing is visible on the marketplace and the plugin is installable.

LGTM!

File: plugins.json [hunk 2]
```diff
@@ -12816,6 +12816,110 @@
...
+    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-servicenow/releases/tag/v2.3.4",
+    "hosting": "",
+    "author_type": "mattermost",
+    "release_stage": "production",
+    "enterprise": false,
+    "signature": "iQIzBAABCAAdFiEExViBuA9p6GO4WtXR0bVLR6XO/sQFAmOi5z0ACgkQ0bVLR6XO/sSXRw/7BG6CQ3YQT6MzdD57Y7xRJBwNGguYQ1h+Rj9E2XDoAt5THgggoYctMonSMQLNK+9lDSFeZe8s7Ezxu3bHTv+fNTyYflEQamV63+RnDtQOU4M+5D8tq8Qbhk80lb/ARA1btt12emfJnZaEuKkIjY+61ZlTuLuS2foRWu8gfsMCqR5d7/fWp9d46gGpAMG40lKSGo8RtCxD5v41ePFA/tj68VSRCA8ZAHMLQXExHG2Rdqq6xeJzaw0XvuiPDC26rQOzJstb2IqznxZERxb0GfYGTuv7txuAKxBKvJ+6IG259EaJbRR+GLgRLP+pMt5S0H6Ezlq0Bn4gUvo/fO68NsZr8AzIBUq9452f4mO4jcMzf1cyWmZ5zCZeYWwfiRdYH4MIjwLtAHjIzRsaRfxnMRtfCXsZfUAcDs5qCpEBHvA5XSn+Nz0Eay+vo0QRYVNG5Y2Mz/UoVu1n9ar8UlXvPz+0y0ZE+kJyv7xTOXiVmsMEC+xhRne8DeJNkR+fwwbrFO+qrg5G+KH2S3pNJp6mmQ0FMw3TYmkDtgkkKS6FTDexRNrr9BskjYx8m9D8vpzE1vLRBWA1Em5h6qoV5mdwqNZQxyJ+sY61BeCKJG2tOw4f/YowWNKw6OX734qbFVUAgPPc6A4Mufy+DUQus4LcuZNrEwqh/UNrAViK3Z8lEBmkOPg=",
+    "repo_name": "mattermost-plugin-servicenow",
+    "manifest": {
+      "id": "mattermost-plugin-servicenow",
+      "name": "Servicenow Plugin",
```
Comment (mickmister) [inline plugins.json:12832]: 0/5, not for this PR

```suggestion
      "name": "ServiceNow Plugin",
```

**Review by mickmister — APPROVED** [review]

File: plugins.json (hunk 2 above)
Comment (mickmister) [reply plugins.json:12832]: Submitted a PR for this https://github.com/mattermost/mattermost-plugin-servicenow/pull/159

Comment (hanzei) [reply plugins.json:12832]: Good catch :+1:

**Pull request merged**